capstone/src/test/java/com/google/univiz/api/SearchResourceImplTest.java

```diff
@@ -0,0 +1,36 @@
+package com.google.univiz.api;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+@RunWith(JUnit4.class)
+public final class SearchResourceImplTest {
+  @Mock private SuggestionDataApi mockSuggestionApi;
+
+  @Rule public MockitoRule mockitoRule = MockitoJUnit.rule();
+
+  @Test
+  public void testGetSuggestions() throws Exception {
+    String partialCollegeName = "Sta";
+    SuggestionData stanford = new SuggestionData("Stanford University", 0);
+    List<SuggestionData> cannedResponse = new ArrayList<>();
+    cannedResponse.add(stanford);
```

Comment:
Leverage [Guava utilities](https://guava.dev/releases/23.0/api/docs/com/google/common/collect/Lists.html#newArrayList-E...-) for brevity.